test/integration/basepath/test/index.test.js

```diff
@@ -32,7 +34,16 @@ jest.setTimeout(1000 * 60 * 2)
 const appDir = join(__dirname, '..')
 
 const runTests = (context, dev = false) => {
-  if (!dev) {
+  if (dev) {
+    it('should render error in dev overlay correctly', async () => {
+      const browser = await webdriver(context.appPort, '/docs/hello')
+      await browser.elementByCss('#trigger-error').click()
+      expect(await hasRedbox(browser)).toBe(true)
+
+      const errorSource = await getRedboxSource(browser)
+      expect(errorSource).toContain('oops heres an error')
```

Comment:
This test needs to capture the stack frame and sure it was resolved.

Reply:
Updated to check the full stack frame